core/services/pipeline/graph.go

```diff
@@ -155,11 +151,18 @@ func (n *TaskDAGNode) String() string {
 	return n.dotID
 }
 
+var bracketQuotedAttrRegexp = regexp.MustCompile(`^\s*<([^<>]+)>\s*$`)
+
 func (n *TaskDAGNode) SetAttribute(attr encoding.Attribute) error {
 	if n.attrs == nil {
 		n.attrs = make(map[string]string)
 	}
-	n.attrs[attr.Key] = attr.Value
+
+	// Strings quoted in angle brackets (supported natively by DOT) should
+	// have those brackets removed before decoding to task parameter types
+	sanitized := bracketQuotedAttrRegexp.ReplaceAllString(attr.Value, "$1")
```

Comment:
How come the dot lib doesn't strip them for us? Can we use the dot native parsing library instead?

Reply:
It seems to think that they're HTML tags, lulz
Which dot native parsing lib?

Reply:
Yeah it's a "html string" intended to pass through HTML as a "html label" https://graphviz.org/doc/info/shapes.html#html